src/main/resources/db/migration/V1.2__Sjp_court_load.sql

```diff
@@ -8,3 +8,5 @@ CREATE TABLE IF NOT EXISTS court (
 INSERT INTO court(court_id, jurisdiction, name)
 VALUES(0, '{}', 'Single Justice Procedure (SJP)')
 ON CONFLICT (court_id) DO NOTHING;
+
+UPDATE court SET name = 'Single Justice Procedure' WHERE court_id = 0;
```

Comment:
ive already updated this in another story, additionally you shouldnt set it then update it, just set it to the right name, but still, ive already done it

Reply:
I've updated this PR slightly now.

V1.1 has been kept as a separate file (rather than renaming), as flyway requires the previous files to still be there after running locally.

V1.2 is now the new file, with a single line that updates V1.1 to be Single Justice Procedure